lib/std/http/Server.zig

```diff
@@ -704,6 +649,13 @@ pub fn accept(server: *Server, options: AcceptOptions) AcceptError!Response {
             },
         },
     };
+
+    resp.connection.buffered = .{ .buffered_reader = .{
+        .unbuffered_reader = &resp.connection,
```

Comment:
`&resp.connection` looks like a stack pointer, are these being invalidated once the function returns?

Reply:
That's how it's handled in Client because it has to be pushed into the connection pool. This probably would have happened eventually to support pooling keep-alive connections on the server so they could be polled.

Reply:
Thanks, had done this change.